src/technologies/h.json

```diff
@@ -1788,11 +1788,38 @@
     "scriptSrc": "/v1/lst/universal-script\\?ph\\=",
     "website": "https://hyros.com"
   },
+  "Hyva Checkout": {
+    "cats": [
+      41
+    ],
+    "description": "A Checkout built for Hyvä Themes using Magewire, Alpine.js and Tailwind CSS.",
+    "js": {
+      "hyvaCheckout": ""
+    },
+    "headers": {
+      "X-Built-With": "^Hyva Checkout$"
+    },
+    "icon": "Hyva Themes.svg",
```

Comment:
Can you add the icon to the repo under `src/images/icon`?

Reply:
It is present already, as it's used for "Hyvä Themes"